Configure

```diff
@@ -455,6 +455,7 @@ my @disablables = (
     "filenames",
     "fips",
     "fips-securitychecks",
+    "fips-ems_check",
```

Comment:
I still feel like this is an overkill to have this as a separate build option.

Reply:
I think we'll want this once the transition comes into force.